server/controllers/tournamentController.ts

```diff
@@ -20,29 +19,22 @@ const getAllTournaments = catchAsync(async (req, res) => {
   const page = Number(req.query.page) || 1;
   const pageSize = Number(req.query.pageSize) || 100;
 
-  const data = await TournamentRepository.getAll(page, pageSize);
-  const enrichedTournaments = await Tournament.enrichTournamentsWithMSUsers(data.tournaments, `Bearer ${req.cookies.jwt}`);
-
+  const enrichedData = await Tournament.getAll(page, pageSize, `Bearer ${req.cookies.jwt}`);
   res.status(200).json({
-    data: {
-      totalRows: data.totalRows,
-      tournaments: enrichedTournaments,
-    },
+    data: enrichedData,
```

Comment:
Nice! I wonder if variable name `data` would be better then `enrichedData` bcz this also includes totalRows